source/write-operations/insert-documents.txt

```diff
@@ -78,26 +71,31 @@ To learn more about document structure and rules, see the
 Insert One Document
 -------------------
 
-To add a single document to a MongoDB collection, call the ``insertOne()``
-method and pass the document you want to add. After you call the
-``insertOne()`` method, chain the ``subscribe()`` method and pass in a subscriber.
+To add a single document to a MongoDB collection, use the static
+``Mono.from`` method from the ``Mono`` ``Publisher`` to call the ``insertOne()``
+method and pass the document you want to add. The ``Mono`` ``Publisher`` is a
```

Comment:
S: recommend breaking this sentence down. Seems like the steps are:
1. use the insertOne method, and pass the document as a parameter. 
2. Pass the insert method call to the Mono.from method (why?)